cmd/test/cmd.go

```diff
@@ -241,26 +235,14 @@ func doRun(ctx context.Context, servicesToTest []string, options *Options, ioCtr
 		}
 	}
 
-	namer := deployCMD.Namer{
-		KubeClient:   kubeClient,
-		Workdir:      cwd,
-		ManifestPath: options.ManifestPathFlag,
-		ManifestName: options.Name,
-	}
-
-	devenvName := options.Name
-	if devenvName == "" {
-		devenvName = namer.ResolveName(ctx)
-	}
+	shouldDeploy := options.Deploy
 
-	namespace := manifest.Namespace
-	if namespace == "" {
-		namespace = okteto.GetContext().Namespace
+	if shouldDeploy && manifest.Deploy == nil {
+		oktetoLog.Warning("Nothing to deploy. The 'deploy' section of your Okteto Manifest is empty. For more information, check the docs: https://okteto.com/docs/core/okteto-manifest/#deploy")
```

Comment:
@pchico83 I am adding a warning if the user set `--deploy` but then has an empty deploy section. Let me know if you would change anything.

Reply:
This also preents the panic that we have now, while accessing `manifest.Deploy.Remote`.

Reply:
`Nothing to deploy` sounds a bit odd but don't know why. I don't have a better alternative. I think we can merge it as it is and change it later if needed